extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -236,8 +230,10 @@ export class Visibility {
   /**
    * @param {!JSONType} config
    * @param {!VisibilityListenerCallbackDef} callback
+   * @param {boolean} visible True if the config applies to the page when
+   *        visible. False when it applies when page is hidden.
    */
-  listenOnce(config, callback) {
+  listenOnce(config, callback, visible) {
     const element = this.win_.document.getElementById(config['selector']
```

Comment:
yes. There is plan to allow other amp- elements by tag name but that isn't implemented yet.